docs/pages/mods-list.mdx

```diff
@@ -0,0 +1,11 @@
+import Image from "next/image";
+
+# Available Mods
+
+
+Some Mods are in development and some others are ones that we're excited for.
+Anyone can [create a Mod](/create-mod/getting-started.mdx), but we're creating some of the first ones while we develop the initial Mod SDKs.
+  
+<Image src="/all-mods.png" alt="all mods" width="600" height="300" />
```

Comment:
is it possible to embed the notion page maybe? would be nice not to have to update this list manually

Reply:
yeah have added a note that we should potentially open up that list